packages/plugin-kit/src/hooks/slice-template-library-read.ts

```diff
@@ -0,0 +1,45 @@
+import type { SharedSlice } from "@prismicio/types-internal/lib/customtypes";
+
+import type {
+	ExtendSliceMachineHook,
+	SliceMachinePluginOptions,
+	SliceMachineHook,
+} from "../types";
+
+/**
+ * Data provided to `slice-template-library:read` hook handlers.
+ */
+export type SliceTemplateLibraryReadHookData = {
+	templateIds?: string[];
+};
+
+/**
+ * Return value for `slice-template-library:read` hook handlers.
+ */
+export type SliceTemplateLibraryReadHookReturnType = {
+	templates: {
+		model: SharedSlice;
+		mocks: string;
+		componentContents: string;
```

Comment:
same as above, maybe "component", but I'm not sure as it could make sense to be obvious that it's the content and not a file path whatsoever 🤔 (but since it's also the content for model/mocks, maybe it's fine/clear also)